anomalib/models/dfm/model.py

```diff
@@ -78,8 +78,8 @@ def validation_step(self, batch, _):  # pylint: disable=arguments-differ
         Similar to the training step, features are extracted from the CNN for each batch.
 
         Args:
-          batch: Dict: Input batch
-          batch_idx: int: Index of the batch.
+          batch (List[Dict[str, Any]]): Input batch
+          _ (int): Index of the batch.
```

Comment:
We could remove this. It'll appear in the documentation, which won't make sense to the reader.